iron-component-page.html

```diff
@@ -192,37 +172,39 @@
       }
 
       .no-docs {
-        @apply(--layout-horizontal);
-        @apply(--layout-center-center);
-        @apply(--layout-fit);
+        @apply --layout-horizontal;
+        @apply --layout-center-center;
+        @apply --layout-fit;
       }
 
       .docs-header {
-        @apply(--layout-flex);
+        @apply --layout-flex;
       }
     </style>
 
     <hydrolysis-analyzer id="analyzer" src="[[_srcUrl]]" transitive="[[transitive]]" clean analyzer="{{_hydroDesc}}" loading="{{_hydroLoading}}"></hydrolysis-analyzer>
     <iron-ajax id="ajax" url="[[docSrc]]" handle-as="json" on-response="_handleAjaxResponse" on-error="_handleError"></iron-ajax>
 
-    <paper-header-panel id="headerPanel" mode="[[scrollMode]]">
-      <paper-toolbar catalog-hidden>
-        <div class="docs-header">
-          <!-- TODO: Replace with paper-dropdown-menu when available -->
-          <select id="active" value="[[active]]" on-change="_handleMenuItemSelected">
-            <template is="dom-repeat" items="[[docElements]]">
-              <option value="[[item.is]]">[[item.is]]</option>
-            </template>
-            <template is="dom-repeat" items="[[docBehaviors]]">
-              <option value="[[item.is]]">[[item.is]]</option>
-            </template>
-          </select>
-        </div>
-        <iron-selector attr-for-selected="view" selected="{{view}}" id="links" hidden$="[[!docDemos.length]]">
-          <a view="docs"><iron-icon icon="description"></iron-icon> Docs</a>
-          <a view="[[_demoView(docDemos.0.path)]]"><iron-icon icon="visibility"></iron-icon> <span>Demo</span></a>
-        </iron-selector>
-      </paper-toolbar>
+    <app-header-layout>
```

Comment:
`has-scrolling-region` or `fullbleed` attribute should be added to `<app-header-layout>`, otherwise the height of `#content` will be 0 and demos will not be visible.